### PR TITLE
refactor: Renamed yaw to azimuth

### PR DESCRIPTION
**Pull request by frgfm**
This PR introduces the following modifications:
- renamed device "yaw" to "azimuth" which is the correct term geometrically speaking
- updated all unittests
- when the alert azimuth is not specified, takes the one of the device
- made alert azimuth non nullable

Any feedback is welcome!

### CODE REVIEW

**Review by fe51 — APPROVED** [review]

File: src/app/api/schemas/devices.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ class DeviceUpdate(Login):
     lat: Optional[float] = Field(..., gt=-90, lt=90, example=44.765181)
     lon: Optional[float] = Field(..., gt=-180, lt=180, example=4.514880)
     elevation: Optional[float] = Field(..., gt=0.0, lt=10000, example=1582)
-    yaw: Optional[float] = Field(..., ge=0, le=360, example=110)
+    azimuth: Optional[float] = Field(..., ge=0, le=360)
```
Comment (fe51) [inline src/app/api/schemas/devices.py:71]: We do not want any example ?

**Comment by frgfm** [comment]
No need to review until I get the unittests to work :)

**Comment by fe51** [comment]
Ho okay sorry, thought it was done ! :)

**Comment by frgfm** [comment]
Now it's all good :)

**Pull request merged**